net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Delayed delivery notification for qmail"
+SRC_URI="http://untroubled.org/qmail-notify/archive/${P}.tar.gz"
+HOMEPAGE="http://untroubled.org/qmail-notify/"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~hppa ~ppc ~sparc ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="
+	${DEPEND}
+	virtual/cron
+	virtual/qmail
+"
+
+src_prepare() {
+	eapply_user
+
+	echo "$(tc-getCC) ${CFLAGS}" > conf-cc
+	echo "$(tc-getCC) ${LDFLAGS}" > conf-ld
+}
+
+src_install () {
+	exeinto /usr/sbin
+	doexe qmail-notify
+
+	exeinto /etc/cron.hourly
+	doexe "${FILESDIR}"/qmail-notify.cron
+
+	dodoc README ANNOUNCEMENT TODO cron.hourly NEWS VERSION
```

Comment:
I have not looked into these files, but VERSION is probably meaningless. Also look if TODO is really worth installing, at least for qmail it really isn't.

Reply:
VERSION and TODO are not worth of installing. How about cron.hourly? It is basically script with the same functionality of the one we install from `${FILESDIR}/qmail-notify.cron`.

Reply:
Your choice as maintainer ;) I just tell you what I have found when renovating netqmail.

Reply:
Ok, thanks!